regolith/tools.py

```diff
@@ -2315,3 +2315,42 @@ def get_appointments(person, appointments, target_grant=None):
             appointments.append((person.get('_id'), bd, ed, appt.get('loading'),
                          round(weighted_duration, 2), appt.get('grant')))
     return appointments
+
+def add_todo_id_to_tasks(gtx, client, db, coll, milestone_id, task_id):
```

Comment:
This needs a test and a docstring.
1) write the docstring first.  It will help you understand what the function is supposed to do
2) then write the tests for expected behavior under different circumstances.
3) then tweak the code so the tests pass.

In general, try and make functions do just one thing.  This function _may_ be doing too much (not sure actually, it may be ok).  If they are doing two things, consider splitting it to two functions where on function calls another.